api/receiver/api.go

```diff
@@ -78,6 +79,24 @@ func (a apiImpl) AddTestRun(testRun *shared.TestRun) (shared.Key, error) {
 	return key, nil
 }
 
+func (a apiImpl) IsAdmin(r *http.Request) bool {
```

Comment:
No tests for this method?

Reply:
Thanks! I should've noticed that this is now stubbed out in all existing tests... I'll add a new test to `api_medium_test.go`.